sci-misc/boinc/boinc-7.6.25.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+#WANT_AUTOMAKE="1.11"
+
+AUTOTOOLS_AUTORECONF=true
+
+inherit autotools-utils eutils flag-o-matic systemd user versionator wxwidgets
```

Comment:
with `EAPI=6`, `autotools-utils.eclass` was deprecated. Just switch back to calling `eautoreconf` in `src_prepare`